tests/prototype/ray_charles.e2e.spec.js

```diff
@@ -25,67 +27,78 @@ function runTests (driver) {
     // Set the implicit wait to 3 seconds
```

Comment:
I can't add a comment in the right place here, but on line 24 of this file, place change "Legacy Ray Charles" to "Ray Charles". Ths ZIM here is a modern ZIM, not a legacy ZIM. (This was my mistake, I know you didn't change that code.)